migration_log.csv

```diff
@@ -0,0 +1,432 @@
+costoptimizationdata317256447485,cid-data-317256447485,is_modified,file_date
```

Comment:
This is not looks like needed

Reply:
Oops. I had my gitignore update in a different branch that I have not pushed yet